data/json/vehicle_parts.json

```diff
@@ -251,14 +251,19 @@
         "broken_symbol" : "#",
         "broken_color" : "brown",
         "durability" : 100,
+        "requirements": {
+          "install": {
+            "time": 60000,
+            "components": [ [ [ "rope_6", 1 ] ] ]
+          }
+        },
         "item" : "frame_wood_light",
         "difficulty" : 0,
         "location" : "structure",
-        "flags" : ["NAILABLE"],
-        "breaks_into" : [
+        "breaks_into" : {
```

Comment:
You need to be developing locally though and checking against the game binary. You're current problem isn't linting - the game wouldn't start with the contents of this PR.